src/DssSpell.sol

```diff
@@ -52,22 +52,159 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant ONE_HUNDREDTH_PCT  = 1000000000003170820659990704;
+    uint256 constant THREE_PCT          = 1000000000937303470807876289;
+    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
 
-    /**
-        @dev constructor (required)
-        @param officeHours true if officehours enabled
-    */
-    constructor(bool officeHours) public DssAction(officeHours) {}
+    constructor() public DssAction() {}
 
     uint256 constant WAD        = 10**18;
     uint256 constant RAD        = 10**45;
     uint256 constant MILLION    = 10**6;
 
+    bytes32 constant ETH_A_ILK          = "ETH-A";
```

Comment:
I think is better we use directly the ilk names everywhere. I don't see the real benefit of using `ETH_A_ILK` instead of `"ETH-A"`. They have the exact grade of description but we have an extra level of indirection to check.

Reply:
I mentioned this the last time you brought this up, but I do prefer having the compiler check when ilks are repeated. Maybe in this case it's not so important as most of these are being used once, but sometimes you have to repeat an ilk several times throughout a spell. Inlining all the strings brings a new danger of typos. That being said, I'm not super strongly opinionated on this, so if you really prefer this switch I'll make it happen. What do you think @brianmcmichael ?

Reply:
Ah sorry didn't see your response last time.
Fair points, this used to happen a lot with collateral onboarding before `dss-exec-lib`. Nowadays the repeating might be 2 times in an `ilk` (debt ceiling + fee), don't think we will have many more in general. So we reduce the amount of lines the code to review. Btw the test should catch  if there is a typo in some `ilk`.

Reply:
Just looking this over again. You're right ilks are only being used once most of the time now. I think you've convinced me. I'll make the switch.

Reply:
I'm kind of in the middle on this one. Every time we use an ilk label as a string we're exposing ourself to some additional homomorph risk, and for efficiency's sake I'd prefer to have named constants when we use something more than once, but in this case I don't feel very strongly about it and would lean in favor of code consistency.